certified-connectors/ITautomate/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "07c09c6f-e066-4fa5-8a49-dfe289854a2f",
```

Comment:
@ITautomate you can remove this one, secret-look-a-like-values are not allowed. Please change it to just an empty string.

Reply:
Fixed